django/poetry.lock

```diff
@@ -1830,1211 +1882,159 @@ testing = ["coverage (>=5.0.3)", "zope.event", "zope.testing"]
 [metadata]
 lock-version = "1.1"
 python-versions = "^3.8"
-content-hash = "0f21f51d22407bc07eac889f2a8b41388ee090d0758108c0fde3d24dc8b57e11"
+content-hash = "d681c64925b1a6f144acb8eae466865371c87ceac6e7661c2dbe0fcea4f7fcbf"
 
 [metadata.files]
 abyss = []
-amqp = [
```

Comment:
What's going on here? I have an ass feeling these shouldn't be removed?

Reply:
Good question and you're probably right; I'll investigate

Reply:
Something indeed was funky; regenerated it and way smaller diff now. So this should be resolved.